cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1705,64 +1735,85 @@ pub(super) mod test {
+            #   # -> Perform initialization steps, e.g. create a python venv
+            #   # -> Useful environment variables:
+            #   #      - FLOX_ENV_PROJECT=/home/user/example
+            #   #      - FLOX_ENV=/home/user/example/.flox/run
+            #   #      - FLOX_ENV_CACHE=/home/user/example/.flox/cache
             # '''
 
-            # Scripts defined in the `[profile]` section are *sourced* by *your shell* and
-            # inherit environment variables set in the `[vars]` section and by `[hook]` scripts.
-            # The `profile.common` script is sourced by all shells and special care should be
-            # taken to ensure compatibility with all shells, after which exactly one of
-            # `profile.{bash,fish,tcsh,zsh}` is sourced by the corresponding shell.
+
+            ## Profile script ----------------------------------------------------
+            ## ... sourced by _your shell_ when you run 'flox activate'.
+            ## -------------------------------------------------------------------
             [profile]
             # common = '''
-            #   echo "it's gettin' flox in here"
+            #   gum style \
+            #   --foreground 212 --border-foreground 212 --border double \
+            #   --align center --width 50 --margin "1 2" --padding "2 4" \
+            #     $INTRO_MESSAGE
             # '''
+            ## Shell specific profiles go here:
+            # bash = ...
+            # zsh  = ...
+            # fish = ...
 
-            # The `[services]` section of the manifest allows you to define services.
-            # Services defined here use the packages provided by the `[install]` section
-            # and any variables you've defined in the `[vars]` section or `hook.on-activate` script.
+
+            ## Services ----------------------------------------------------------
+            ##  $ flox services start             <- Starts all services
+            ##  $ flox services status            <- Status of running services
+            ##  $ flox activate --start-services  <- Activates & starts all
+            ## -------------------------------------------------------------------
             [services]
-            # postgres.command = "postgres --config-file=pg.conf"
+            #myservice.command = "python3 -m http.server"
+
 
-            # Additional options can be set in the `[options]` section. Refer to
-            # manifest.toml(5) for a list of available options.
+            ## Other Environment Options -----------------------------------------
             [options]
-            systems = []
+            # Systems that environment is compatible with
+            systems = [
+              "aarch64-darwin",
+              "aarch64-linux",
+              "x86_64-darwin",
+              "x86_64-linux",
+            ]
             # Uncomment to disable CUDA detection.
-            # cuda-detection = false
+            #cuda-detection = false
```

Comment:
```suggestion
            # cuda-detection = false
```

all other commented lines use a space after the `#` which i think also aligns with the toml style guide (by way of example, not the spec explicitly saying so)

Reply:
👍🏼 makes sense, it does look inconsistent, good catch.